html_sanitizer.rst

```diff
@@ -793,17 +793,17 @@ URLs of ``<a>`` elements:
                 ->sanitizer('app.post_sanitizer')
                     // if `true`, all URLs using the `http://` scheme will be converted to
                     // use the `https://` scheme instead. `http` still needs to be
-                    // allowed in `allowedLinkSchemes`
+                    // allowed in `allowLinkSchemes`
                     ->forceHttpsUrls(true)
 
                     // specifies the allowed URL schemes. If the URL has a different scheme, the
                     // attribute will be dropped
-                    ->allowedLinkSchemes(['http', 'https', 'mailto'])
+                    ->allowLinkSchemes(['http', 'https', 'mailto'])
```

Comment:
For the DI configuration this is not the correct change, right?

Reply:
indeed. That's why the CI fails